radicle/src/cob/patch.rs

```diff
@@ -403,21 +409,22 @@ impl Revision {
     }
 }
 
+/// Patch state.
 #[derive(Debug, Clone, Copy, Default, PartialEq, Eq, PartialOrd, Ord, Serialize, Deserialize)]
 #[serde(rename_all = "camelCase", tag = "status")]
 pub enum State {
-    #[default]
-    Proposed,
-    Draft,
     Archived,
+    Draft,
+    #[default]
+    Open,
```

Comment:
The order of the variants matters, since we are using this in the CRDT and there's an `Ord` instance. It's important not to change it unless there's a reason :)

Reply:
83d15e4